src/db/migration/1610660009057-CreateExternalId.ts

```diff
@@ -0,0 +1,30 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateExternalId1610660009057 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+
+        // Creates primary key + index named "external_id_pkey"
+        // Creates a unique constraint named "external_id_external_id_external_id_type_key" and btree index on (external_id, external_id_type)
+        queryRunner.query(
+            `CREATE TABLE IF NOT EXISTS external_id (
```

Comment:
just thinking out loud:  would there be any benefit to keeping the sql in a separate file (and read it in and pass the content into queryRunner)?

Reply:
I think there's a benefit and a drawback. The benefit is that if you use put it in a `.sql` file, then you can maybe get IDE help with syntax checking and such. The drawback is that you still have to create the migration and update the corresponding entity, so it's yet another thing you have to do to write a migration. I think it could be overall beneficial, but maybe we should wait and see if the lack of IDE help with sql turns out to be a pain point after all.